### DISCUSSION TITLE
Domínio da Python Nordeste

### DISCUSSION THREAD

**Discussion by cecivieira** [apyb/comunidade#70] [Geral]
Oi povo,

Vocês sabem dizer quem é que gerencia o domínio da Python Nordeste? Estamos com problema:

![image](https://user-images.githubusercontent.com/30605862/213941594-1a333d25-9695-47f1-a972-169f490d7984.png)

cc @lipemorais

**Comment by rougeth** [comment]
Não sei quem cuida desse domínio, mas ele não esta com a APyB.

**Reply by alynnefs** [reply]
Quem cuida do domínio é o @fabiocerqueira

**Comment by fabiocerqueira** [comment]
Obrigado pelo aviso, desculpa pela falta de atencao com dominio, ele ta' renovado e verificado, deve voltar em breve

**Comment by cecivieira** [comment]
Obrigada, @fabiocerqueira !!

**Comment by felipythondev** [comment]
Muito obrigado, @fabiocerqueira pela prontidão.

@rougeth não seria o caso de passar esse domínio para a APyB?

**Reply by rougeth** [reply]
Conversamos na última reunião sobre isso. A APyB pode sim cuidar do domínio da PyNE.

**Comment by fabiocerqueira** [comment]
Chequei agora e a verificacao ja' foi propagada, tudo de volta ao normal.

Sobre transferencia do dominio para APyB, se for de interesse da organizacao, eu estou disposto a colaborar :D

**Comment by cecivieira** [comment]
@fabiocerqueira @rougeth Podemos começar a transferência do domínio pra APyB? Como faz?

**Comment by fabiocerqueira** [comment]
O processo para transferir e' bem simples: https://support.dnsimple.com/articles/transferring-domain-away/

Atualmente o dominio ta' no dnsimple e as configuracoes no cloudflare, quem seria o ponto de contato para fazer a transferencia do lado da APyB?

**Reply by cecivieira** [reply]
acho que @rougeth é o ponto de contato do lado da APyB

**Comment by thaty-maru** [comment]
Olá, pessoal
Sou Thatianne, e estou organizando a Python Norte 2023. Gostaria de uma ajuda para adquirir o domínio nos padrões dos eventos de Python. Se puderem me avisar agradeço muito 🙏🏼

**Reply by cecivieira** [reply]
@rougeth tu sabe se é possível um domínio pra Python Norte, similar ao que temos na Nordeste?

**Comment by tiidadavena** [comment]
@fabiocerqueira você teria como add este [CNAME](https://pyne2023.onrender.com/) das configurações do Cloudfare? Se achares melhor, podemos ver para transferir a conta do Cloudfare para o email da PyNE. O motivo é que o Netlify quebrou o layout do redesign do site e migramos para uma nova ferramenta de deploy.

https://render.com/docs/configure-cloudflare-dns
